tsconfig.tsbuildinfo

```diff
@@ -0,0 +1,1085 @@
+{
```

Comment:
I don't think this file should be part of the git repo, should it?  It'll change every time somebody runs a build.